scripts/bot-deployment/DeployBotGCP.sh

```diff
@@ -3,7 +3,8 @@ set -e
 
 if [ $# -ne 1 ]; then
     echo "Incorrect number of arguments supplied! First and only argument is bot's name. From this the bot's config and service account will be inferred."
-    echo "example: ./DeployBotGCP.sh ethbtc-monitor-bot"
+    echo "example: ./DeployBotGCP.sh ethbtc-mainnet-monitor"
```

Comment:
change made to be consistent with @nicholaspai request

Reply:
Awesome thanks! nit: you mispelled "available" in line below

Reply:
fixed

Reply:
looks good